pkg/downloader/manager.go

```diff
@@ -459,22 +461,30 @@ func (m *Manager) parallelRepoUpdate(repos []*repo.Entry) error {
 // repoURL is the repository to search
 //
 // If it finds a URL that is "relative", it will prepend the repoURL.
-func findChartURL(name, version, repoURL string, repos map[string]*repo.ChartRepository) (string, error) {
+func findChartURL(name, version, repoURL string, repos map[string]*repo.ChartRepository) (url, username, password string, err error) {
 	for _, cr := range repos {
 		if urlutil.Equal(repoURL, cr.Config.URL) {
-			entry, err := findEntryByName(name, cr)
+			var entry repo.ChartVersions
+			entry, err = findEntryByName(name, cr)
 			if err != nil {
-				return "", err
+				return
```

Comment:
Since the *findChartURL* function now returns named parameters, there's actually no need to specify the returned values. Before my change, the method returned:
```
(string, error)
```
Now it returns:
```
(url, username, password string, err error)
```
So the value of *err* is returned anyway. 
Please let me know if you'd like me to change this.

Reply:
Oh I didn't catch that. Thank you!